api/maven-api-core/src/main/java/org/apache/maven/api/Type.java

```diff
@@ -22,7 +22,7 @@
 import org.apache.maven.api.annotations.Immutable;
 
 /**
- * An artifact's{@code Type} represents a known kind of artifacts.
+ * A dependency's {@code Type} represents a known kind of dependencies.
  * Such types are often associated to an extension and possibly
```

Comment:
associated with

Is this often or always?

Reply:
Always.